docs/source/use_cases/index.rst

```diff
@@ -81,6 +81,16 @@ For example, ``gangreen`` is a registered farmer ``tomato`` asset creator, he se
 
 We simplified asset creation to just a single command ``CreateAsset`` without the need to create complex smart contracts. One the major advantages of Hyperledger Iroha is in its ease, that allows developers to focus on the provided value of their applications. 
 
+Fund management
+--------------
+
+With the support of multisignature transaction it is possible to maintain a fund with multiple number of managers. In that scheme investment can only be done after the confirmation of the quorum participants.
+
+Example
+^^^^^^^
+
+The fund assets should be help at one account. Its signatories should be fund managers, who are dealing with investments and portfolio distributions. That can be added via ``AddSignatory`` command.  All of the assets should be held within one account, which signatories represent the fund managers. Thus the concrete exchanges can be performed with the multisignature transaction, so that everyone will decide on a particular financial decision. The one may confirm a deal with sending the original transaction and his or her signature. Iroha will maintain the transaction sending, so that the deal will not be completed until it is receive the required number of confirmation, that parametrized with the transaction quorum parameter.
```

Comment:
Such a big paragraph worth to be commented once more about the need of newline separators between sentences :)

Reply:
The fund assets should be help at one account.  - the idea is not clear :(

Reply:
Consider removing commas before "so" (two occurrences).

Reply:
the deal will not be completed until it **receives** the required number of confirmation